src/bundles/files/utils.js

```diff
@@ -315,3 +316,25 @@ export const ensureMFS = (store) => {
     throw new Error('Unable to perform task if not in MFS')
   }
 }
+
+/**
+ * Get bytes from file
+ * @param {FileStream} file
+ * @returns {Promise<Uint8Array>}
+ */
+export const fileToByteArray = (file) => {
```

Comment:
You can simplify this by using the [Blob.arrayBuffer()](https://developer.mozilla.org/en-US/docs/Web/API/Blob/arrayBuffer) function which already returns a promise with an ArrayBuffer which you can convert to a Uint8Array.

Though we probably want to stream the upload, assuming that the kubo-rpc-client supports it.